getGit.tex

```diff
@@ -30,7 +30,7 @@
 
 % set fonts
 \usepackage{fontspec}
-\setsansfont{Fontin Sans}
+%\setsansfont{Fontin Sans}
```

Comment:
I think we should leave this uncommented for those of us who are currently able to build with Fontin Sans. The easiest way to fix that will probably be to:
- uncomment this line.
- commit and push to this branch (any new commit on this branch will show up automatically in this PR)
- either fix the environment issue that's breaking Fontin Sans for you, _or_ re-comment that line so you can build the deck, making sure to leave that change out of future PRs (manually, or by committing chunks)